### PR TITLE
Improve mwan rule form

### PR DESCRIPTION
**Pull request by gsanchietti**
Changes:
- clean form after save
- mimic firewall rules form for a better user experience

Issue: NethServer/nethsecurity#671

### CODE REVIEW

File: public/i18n/en/translation.json [hunk 1]
```diff
@@ -970,16 +970,14 @@
       "protocol_udp": "UDP",
       "protocol_icmp": "ICMP",
       "protocol_all": "All",
-      "address_options": {
-        "any": "Any",
-        "address": "Address",
-        "object": "Object"
-      },
+      "address_options_address": "Enter one address",
```
Comment (andre8244) [inline public/i18n/en/translation.json:973]:
```suggestion
      "address_options_address": "Enter an address",
```

File: src/composables/useRuleForm.ts [hunk 2]
```diff
@@ -101,14 +101,20 @@ export function useRuleForm(policies: Ref<Policy[]>, rule?: Ref<Rule | undefined
...
-  const srcType = ref<AddressType>('any')
-  const dstType = ref<AddressType>('any')
+  const dstAddressOptions: { id: AddressType; label: string }[] = [
+    { id: 'address', label: t('standalone.multi_wan.address_options_address') },
+    { id: 'any', label: t('standalone.multi_wan.address_options_any_dst') },
+    { id: 'object', label: t('standalone.multi_wan.address_options_object') }
+  ]
+
+  const srcType = ref<AddressType>('address')
+  const dstType = ref<AddressType>('address')
```
Comment (andre8244) [inline src/composables/useRuleForm.ts:117]: Are we changing the default value from `any` to `address`?

**Review by andre8244 — APPROVED** [review]
If we want to change the default value of source and destination, LGTM

File: src/composables/useRuleForm.ts (hunk 2 above)
Comment (gsanchietti) [reply src/composables/useRuleForm.ts:117]: Yes, now it's the same of firewall rules

File: public/i18n/en/translation.json (hunk 1 above)
Comment (gsanchietti) [reply public/i18n/en/translation.json]: Fixed

**Review by Tbaile — APPROVED** [review]

**Pull request merged**